recruiter/recruiterSignup.php

```diff
@@ -0,0 +1,117 @@
+<?php
+  // Database connection variables
+  require_once('../connectVars.php');
+
+  // Insert the page header and navbar
+  $page_title = 'Recruiter Signup';
+  require_once('./templates/header.php');
+  require_once('./templates/navbar.php');
+
+  //Connect to the database
+  $dbc = mysqli_connect(DB_HOST, DB_USER, DB_PASSWORD, DB_NAME);
+  
+    if(isset($_POST['submit'])){
+      // Grab the sign in data from the post
+      $company_id = mysqli_real_escape_string($dbc, trim($_POST['company-id']));
+      $company_name = mysqli_real_escape_string($dbc, trim($_POST['company-name']));
+      $company_category = mysqli_real_escape_string($dbc, trim($_POST['company-category']));
+      $hr_name = mysqli_real_escape_string($dbc, trim($_POST['hr-name']));
+      $hr_email = mysqli_real_escape_string($dbc, trim($_POST['hr-email']));
+      $password = mysqli_real_escape_string($dbc, trim($_POST['pwd']));
+      $verify_password = mysqli_real_escape_string($dbc, trim($_POST['confirm-pwd']));
+      if(!empty($company_id) && !empty($company_name) && $company_category!="0" &&
+        !empty($hr_name) && !empty($hr_email) && 
+        !empty($password) && !empty($verify_password) &&
+        ($verify_password == $password)){
+          // Check if company_id is available
+          $query = "SELECT * FROM recruiters WHERE company_id = '$company_id'";
+          $data = mysqli_query($dbc, $query);
+          if(mysqli_num_rows($data) == 0){
+            // company_id is available
```

Comment:
*not available

Reply:
This is the column that tells that company id is not already used, so it's available.